plugins/.meta

```diff
@@ -0,0 +1,15 @@
+{
+  "projects": {
+    "job-scheduler": "git@github.com:opensearch-project/job-scheduler.git",
+    "anomaly-detection": "git@github.com:opensearch-project/anomaly-detection.git",
+    "alerting": "git@github.com:opensearch-project/alerting.git",
+    "asynchronous-search": "git@github.com:opensearch-project/asynchronous-search.git",
+    "index-management": "git@github.com:opensearch-project/index-management.git",
+    "k-nn": "git@github.com:opensearch-project/k-nn.git",
+    "notebooks": "git@github.com:opensearch-project/dashboards-notebooks",
```

Comment:
May be I am missing something here, is this file intended to only have OpenSearch plugins and not Dashboards plugins?

Reply:
Correct. I made 2 meta projects, `plugins/.meta` and `dashboards-plugins/.meta`, two sets of plugins.

Reply:
Note that some repos have both an opensearch plugin and a opensearch dashboards plugin, e.g.  `notebooks`.

Reply:
But dashboard plugins are part of `plugins/.meta` is what confused me.
Example: `"notebooks": "git@github.com:opensearch-project/dashboards-notebooks",`

Reply:
That's because the OpenSearch plugin is in that repo, too: https://github.com/opensearch-project/dashboards-notebooks/tree/main/opensearch-notebooks. So the real question is whether we want 2 lists or 1, especially given that there will be duplication.

Reply:
Because it's confusing and just easier to deal with repo names (rather than plugin names), I combined these and made the folder match the repo name.